templates_ark/example_neighborhood_analysis_script.ipynb

```diff
@@ -1,18 +1,34 @@
 {
```

Comment:
<p>Is there any way we can adjust the colorbar width so it doesn't get too close to the heatmap?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/714/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

Reply:
This is an issue with all places where `visualize.draw_heatmap` is called. I noticed it when testing the cell clustering notebook. I believe it is also used in the spatial enrichment notebook. I think we can leave this for another pull request, so whoever makes the fix can confirm the heatmaps look okay in all of those notebooks.

Reply:
I also noticed just now that some of the other example notebooks do not have example output visualizations, as you have requested for this one. I think the pixel clustering and cell clustering notebooks, and spatial enrichment notebooks would also benefit from example outputs.

Reply:
@cliu72 seems like the sample visualizations I had in the pixel and cell clustering notebooks got removed during some recent PRs, I'll ask @srivarra put those back in while he addresses the example dataset PR on his end.

Reply:
> This is an issue with all places where `visualize.draw_heatmap` is called. I noticed it when testing the cell clustering notebook. I believe it is also used in the spatial enrichment notebook. I think we can leave this for another pull request, so whoever makes the fix can confirm the heatmaps look okay in all of those notebooks.

Yeah, the underlying `heatmap` function unfortunately doesn't offer great documentation in order to fix this. I think it'll require looking under the hood to see what internal properties are being set, I had to do that with a different `seaborn` function somewhere.

Reply:
@srivarra yes, the overlays for both notebooks as well as the weighted channel heatmaps for the cell clustering notebook.